package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "start": "node start.js",
     "lint": "standard",
-    "test": "npm run lint"
+    "test": "gulp test && npm run lint"
```

Comment:
Travis is running the test suite twice, because Travis.yml is doing:

```
script:
- gulp test
- npm test
```

But `npm test` already runs `gulp test`. I would suggest updating Travis.yml to only run `npm test`